versions.json

```diff
@@ -1,4 +1,5 @@
 {
-    "0.3.3": "0.9.16",
-    "0.5.0": "0.11.13"
-}
\ No newline at end of file
+  "0.3.3": "0.9.16",
+  "0.5.0": "0.11.13",
+  "0.6.0": "0.13.0"
```

Comment:
I'll not positive the newest version works in 0.12.X, I can double check though.

Reply:
I think it's ok if we release it for 0.13.X!